samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/util/BeanDateComparator.java

```diff
@@ -111,6 +111,11 @@ public int compare(Object o1, Object o2)
         i2 = sdf.parse(s2);
         break;
       } catch (ParseException e) {
+          try {
+              i2 = sdf.parse(s2);
```

Comment:
I don't understand. The instructor is using two different date formats? Or is s1 empty?

Reply:
s1 is empty.

![image](https://user-images.githubusercontent.com/15141743/207243773-3ea5127e-58d0-430b-b208-b98b52f6f45d.png)